javascript/javascript2/week3/stringToLog.html

```diff
@@ -0,0 +1,54 @@
+<head>
+    <title>stringToLog</title>
+    <meta name="viewport" content="width=device-width, initial-scale=1" />
+
+</head>
+
+<body>
+    <main>
+        <button id="mybtn">Click me!</button>
+
+
+
+
+
+    </main>
+    <script>
+        let btn = document.getElementById("mybtn");
+        let script = document.createElement("script");
+        script.type = "text/javascript";
+
+
+        function delaying(delay, stringToLog) {
+            setTimeout(() => {
+
+                console.log(stringToLog);
+            }, delay * 1000);
+
+        }
+
+
+        btn.addEventListener('click', delaying(5, "Called after 5 seconds"));
```

Comment:
You need to call this function only after clicking on the button.
I would advise to use callback, like this
` btn.addEventListener('click', () => delaying(5, "Called after 5 seconds"));`